modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -201,6 +215,16 @@ The application listing component shows applications that use an API.
 
 // check *New Application URL* description
 
+== Learn More Button
+
+The learn more button enables users to access a CMS page with additional information about an API.
```

Comment:
enables users to access a custom page with additional information about the API.

Reply:
Fixed. Thank you!